### ISSUE TITLE
How to balance mcool files.

### ISSUE THREAD

**Issue by 1944498970** [tanjimin/C.Origami#36]
Hi，jimin
    How do you perform balance processing on mcool files? When I use hic-bench's track to generate an mcool file using juicertools, I encounter an error because the -k parameter in addnorm is not available in the new version. Then I use Cooler to balance the mcool file, but the generated values are extremely small, mostly ranging from 0 to 0.01. However, when I downloaded your data and viewed it, I found that the values were distributed from 0 to 40.
 Thanks!

**Comment by BenxiaHu** [comment]
Hi jimin,
    Just want to confirm the input mcool should be balanced, right?
thanks!

**Comment by tanjimin** [comment]
Yes it should be balanced. Another important step is restoring to the scale of original count level (you can multiply by a large number), otherwise log1p won't work.

**Comment by BenxiaHu** [comment]
thank you!
restoring to the scale of original count level ----do you mean I don't used the scaled balanced matrix? 

I used cooler to generate mcool file. cooler automatically balance and scale the matrix. so the sum of each row/column is 1.
I used your C.Origami and the mcool file generate by cooler to train a model. I don't see any error and get the model.

please advise.
